biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -754,8 +757,12 @@ private void doSecurity() {
 
 	@SuppressWarnings("deprecation")
 	private boolean isFragment(Bundle b) {
-		return padmin != null
-				&& padmin.getBundleType(b) == org.osgi.service.packageadmin.PackageAdmin.BUNDLE_TYPE_FRAGMENT;
+		if(padmin != null){
+			return padmin.getBundleType(b) == PackageAdmin.BUNDLE_TYPE_FRAGMENT;
```

Comment:
The launcher is built against Core 4.3.1 which includes the wiring API (i.e. BundleRevision). So just remove the use of PackageAdmin entirely.